### DISCUSSION TITLE
Hard to drag the 3d object down

### DISCUSSION THREAD

**Discussion by jajaghu** [google/model-viewer#1939] [Q&A]
### Description

as I posted on my twitter, I found that this model-viewer latest library hard to drag the object (3d model) down, also its hard for me to track the surface. This is my website https://ancientjava.com/arsip-candi/relief-candi-jago

early model-viewer I used so smooth to drag down the 3d object to the surface ground. Latest update better in loading object faster than early library but lack in dragging 3d  model down. I test on my samsung J7 Pro and Samsung S 9

**Comment by elalish** [comment]
I still have no idea what you mean by "drag the 3d object down". Are you talking about 3D or AR mode? Are you talking about page scroll or model pitch? Or something else entirely? Perhaps you could post a screen recording of the good vs bad?

**Comment by jajaghu** [comment]
> I still have no idea what you mean by "drag the 3d object down". Are you talking about 3D or AR mode? Are you talking about page scroll or model pitch? Or something else entirely? Perhaps you could post a screen recording of the good vs bad?

Hello,
I made 2 video from early arcore release and a new one [https://bit.ly/3mY5t9S](https://bit.ly/3mY5t9S)

**Comment by elalish** [comment]
Thanks for the examples. The main difference is the first example used the scene-viewer AR mode while the second used webxr mode (we changed the default, but you can still select the one you want). However, the scaling and placement behavior are pretty similar between the two. The main different I saw in the videos was the models used and the scene you were trying to place them on. The first video had a much easier surface for ARCore to track, so that may have been most of the difference. Also, your models are huge (I notice you scaling them down so you can see them properly). You'll probably have an easier experience if you bake a scale into them to fit nicely in a room.

**Comment by jajaghu** [comment]
> Thanks for the examples. The main difference is the first example used the scene-viewer AR mode while the second used webxr mode (we changed the default, but you can still select the one you want). However, the scaling and placement behavior are pretty similar between the two. The main different I saw in the videos was the models used and the scene you were trying to place them on. The first video had a much easier surface for ARCore to track, so that may have been most of the difference. Also, your models are huge (I notice you scaling them down so you can see them properly). You'll probably have an easier experience if you bake a scale into them to fit nicely in a room.

thanks man!, so to change the default simply change to this? `ar-modes=”scene-viewer webxr quick-look` right, actually am also trying to the same surface (floor) but the result still the same. Guess my first problem change the default and secondly bake the model into smaller to fit,

**Comment by jajaghu** [comment]
> Thanks for the examples. The main difference is the first example used the scene-viewer AR mode while the second used webxr mode (we changed the default, but you can still select the one you want). However, the scaling and placement behavior are pretty similar between the two. The main different I saw in the videos was the models used and the scene you were trying to place them on. The first video had a much easier surface for ARCore to track, so that may have been most of the difference. Also, your models are huge (I notice you scaling them down so you can see them properly). You'll probably have an easier experience if you bake a scale into them to fit nicely in a room.

Hi, man. I just changed my default from webxr to Ar scene viewer, that's pretty cool. But I found a small problem. when I open my model in "my space", some of model force closed and returns to web view. my 3d model ranges from 8-9 mb. It's okay or too big for web arcore.

![photo_2020-12-08_16-05-39](https://user-images.githubusercontent.com/70875290/101462901-5418da00-396f-11eb-8d47-405901e90985.jpg)

**Comment by elalish** [comment]
Yes, we use webXR as the default because it's fast and consistent. When you choose Scene-viewer instead, you're opening a separate native app that has to redownload the model and parse and render it using a different renderer. So you may not get consistent results between them. It sounds like scene-viewer may have choked on that model while it works in web mode. Also, what is the image about?
